build.cake

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 // TOOLS
 ///////////////////////////////////////////////////////////////////////////////
-#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.1.20160523.144749
+#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.2.20160623.172854-eap5"
```

Comment:
Yeah, but that is why I thought I would use the latest stable, rather than the prerelease.